bpf/bpf_overlay.c

```diff
@@ -289,6 +295,7 @@ static __always_inline int handle_ipv4(struct __ctx_buff *ctx,
 	bool decrypted;
 	bool __maybe_unused is_dsr = false;
 	int ret;
+	int is_delivered_via_stack = 0;
```

Comment:
```suggestion
	bool is_delivered_via_stack = false;
```

Let's be explicit about this being a boolean value (also in other parts of the patch).